docs/swarm/services.md

```diff
@@ -0,0 +1,252 @@
+<!--[metadata]>
++++
+title = "Deploy services to a swarm"
+description = "Deploy services to a swarm"
+keywords = ["guide, swarm mode, service"]
+[menu.main]
+identifier="services-guide"
+parent="engine_swarm"
+weight=15
++++
+<![end-metadata]-->
+
+# Deploy services to a swarm
+
+When you are running Docker Engine in swarm mode, you run
+`docker service create` to deploy your application in the swarm. The swarm
+manager accepts the service description as the desired state for your
+application. The built-in swarm orchestrator and scheduler deploy your application to nodes in your swarm to achieve and maintain the desired state.
+
+For an overview of how services work, refer to [How services work](how-swarm-mode-works/services.md).
+
+This guide assumes you are working with the Docker Engine in running in swarm
+mode. You must run all `docker service` commands from a manager node.
+
+If you haven't already, read through [Swarm mode key concepts](key-concepts.md)
+and [How services work](how-swarm-mode-works/services.md).
+
+## Create a service
+
+To create the simplest type of service in a swarm, you only only need to supply
+a container image:
+
+```bash
+$ docker service create <IMAGE-ID>
+```
+
+The swarm orchestrator schedules one task on an available node. The task invokes  a container based upon the image. For example you could run the following command to create a service of one instance of an nginx web server:
+
+```bash
+$ docker service create --name my_web nginx
```

Comment:
This is intended to demonstrate the most simple command. I've added your example to publish port and added a link here: 

To make the web server acceisble outside the swarm, you need to [publish the port](#publish-ports-externally-to-the-swarm).